Add missing errors to our API docs and revamp the look of error tables for increased readability (#279)
alphagov/notifications-tech-docs: source/documentation/error_tables/_get_a_page_of_received_text_messages.md

> There are no errors specific to this endpoint. Still, you may also encounter:

Author: If we have a comma, we don't need the full stop here, right?

> There are no errors specific to this endpoint. Still, you may also encounter:

Reviewer: I must have forgotten to delete :(

> There are no errors specific to this endpoint. Still, you may also encounter:

Reviewer: ```suggestion
There are no errors specific to this endpoint, but you may encounter:
```
